File: app/src/main/java/ru/roscha_akademii/medialib/viewvideo/view/ShowVideoView.java
package ru.roscha_akademii.medialib.viewvideo.view;

import com.hannesdorfmann.mosby.mvp.MvpView;

import ru.roscha_akademii.medialib.net.model.Video;

public interface ShowVideoView extends MvpView {
    void showVideo(Video video);
}

File: app/src/main/java/ru/roscha_akademii/medialib/video/playvideo/viewvideo/view/ShowVideoView.kt
package ru.roscha_akademii.medialib.video.playvideo.viewvideo.view

import com.arellomobile.mvp.MvpView
import ru.roscha_akademii.medialib.book.model.local.entity.BookFile

interface ShowVideoView : MvpView {
    fun showVideo(url: String)

    fun showStatus(url: String, title: String?)

    fun showDescription(desc: String?)
}
